app/Website/src/Http/Requests/ContactFormRequest.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace App\Website\Http\Requests;
+
+use Illuminate\Contracts\Validation\ValidationRule;
+use Illuminate\Foundation\Http\FormRequest;
+use Illuminate\Validation\Rule;
+
+class ContactFormRequest extends FormRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     */
+    public function authorize(): bool
+    {
+        return true;
+    }
+
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array<string, ValidationRule|array<mixed>|string>
+     */
+    public function rules(): array
+    {
+        $rules = [
+            'first_name' => 'required|string|max:255',
+            'last_name' => 'required|string|max:255',
+            'email' => 'required|email:filter',
+            'subject' => 'required|string|max:255',
+            'message' => 'required|string|max:1000',
```

Comment:
Please use the array syntax instead of the string syntax for the validation rules to make them more readable.